parm/archive/master_gfs.yaml.j2

```
@@ -3,18 +3,6 @@
 {% set cycle_YMD = current_cycle | to_YMD %}
 {% set cycle_YMDH = current_cycle | to_YMDH %}
 
```

Alternatively, you could just add the check to the masters and convert `int`s to `list`s.

```suggestion

{% if IAUFHRS is integer %}
    {% set IAUFHRS = [IAUFHRS] %}
{% endif %}

```